scripts/create-minikube.sh

```diff
@@ -34,20 +34,34 @@ minikube config set WantUpdateNotification false
 minikube config set WantReportErrorPrompt false
 minikube config set WantNoneDriverWarning false
 minikube config set vm-driver none
+minikube config view
+
+# Setup Docker according to https://kubernetes.io/docs/setup/production-environment/container-runtimes/#docker
```

Comment:
Why do we need to setup docker? Do we use it for building images?

Reply:
Minikube deploys the kubernetes components using Docker (`vm-driver=none`).